web/packages/teleport/src/TopBar/TopBar.tsx

```diff
@@ -191,29 +173,37 @@ export function TopBar({ CustomLogo }: TopBarProps) {
                 </NavigationButton>
               );
             })}
-          </>
-        )}
+          </Flex>
+        </>
+      )}
+      {feature?.hideFromNavigation && (
+        <ButtonIconContainer onClick={handleBack}>
+          <ArrowLeft size="medium" />
+        </ButtonIconContainer>
+      )}
```

Comment:
what does this do?

Reply:
This is shown when using TAG. We remove all the navigation buttons and replace it with this (previously existing) arrow back component. this keeps the current functionality of Access Graph